Fix:

warning: '_btco_aewan_helpdlg_h' is used as a header guard here, followed by #define of a different macro [-Wheader-guard]

https://sourceforge.net/p/aewan/bugs/14/
--- helpdlg.h
+++ helpdlg.h
@@ -33,7 +33,7 @@ programa, armazenada no arquivo COPYING).
 
 
 #ifndef _btco_aewan_helpdlg_h
-#define _btco_aewan_helprdlg_h
+#define _btco_aewan_helpdlg_h
 
 /* invokes the "help" dialog */
 void show_help_dlg(void);
